examples/all-clusters-app/linux/AllClustersCommandDelegate.cpp

```diff
@@ -36,6 +36,12 @@ void AllClustersCommandDelegate::OnEventCommandReceived(const char * json)
 {
     Json::Reader reader;
 
+    if (!mJsonValue.empty())
```

Comment:
Since the provided json is not getting buffered, what happens to it if we're busy processing the previous one? Doesn't it get lost?

Seems like that wouldn't be acceptable right?

Reply:
Yes, as the description said, we only support to process the event command one by one, the current one will get lost if the previous one is still in process. We might be unlikely fall under this scenario considering all command are sent via the command line manually.
  
I can adjust the design to remove this limitation.

Reply:
How is the serialized execution guaranteed? Is there an 'ACK' or 'response' back to the initiator of the command to tell them to proceed to the next command?

Reply:
I have changed the design, and allocated a handler for each incoming json command. Those handlers are allocated dynamically and released after processing. If a new json command come before the previous one in process, it will get queued instead of ignore like before.